minio/minio
# Add time info to health data

## Description

Add the current time from all the nodes, along with the roundtrip time required
to fetch it to the health report data, so that this can be used to detect NTP
configuration issues by `health-analyzer.
  
Return the data under `sys -> config -> time-info`

Sample data of the `config` node:

```json
          "config": { 
            "rlimit-max": 1048576,
            "time-info": {
              "current_time": "2022-01-05T16:15:31.445509567Z",
              "roundtrip_duration": 48
            }
          }
```

## Motivation and Context

To detect if NTP is not configured and alert the user about the same.

## How to test this PR?

- Create a minio cluster involving nodes that have different system time (delta of > 10sec)
- Generate health report for this cluster using `mc admin subnet health {alias} --airgap`
- Verify that the health report JSON contains a new node `time-info` under the `sys -> config` section for each of the servers

## Types of changes

## Checklist:
